docs/support.rst

```diff
@@ -24,5 +24,32 @@ GPG Key Fingerprint: 6D5B EF9A DD20 7580 5747  B70F 9F88 FB52 FAF7 B393
 The public key can be fetched from any GPG keyserver, but be careful: you must
 use the **full fingerprint** to check that you got the correct key.
 
-`Releases <https://github.com/borgbackup/borg/releases>`_ are signed with this GPG key,
-please use GPG to verify their authenticity.
+Verifying signed releases
+-------------------------
+
+`Releases <https://github.com/borgbackup/borg/releases>`_ are signed with the same GPG key and a .asc file is provided for each binary.
+
+To verify a signature, the public key needs to be known to GPG. It can be imported into the local keystore from a keyserver with the fingerprint:
+
+      gpg --recv-keys "6D5B EF9A DD20 7580 5747 B70F 9F88 FB52 FAF7 B393"
```

Comment:
does that work with embedded spaces?

Reply:
It does, when quoted. And since the fingerprint is on the website with the spaces, it was easier to quote than to write 'and remove all spaces manually' :-P

Reply:
no, it's nice, just wondered if it works.